<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputModel</class>
 <widget class="QWidget" name="inputModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="listSplitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="gridLayoutWidget">
      <layout class="QVBoxLayout" name="atmo">
       <item>
        <widget class="QLabel" name="atmoLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <italic>true</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Atmosphere</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="atmoMode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="placeholderText">
          <string>Choose model...</string>
         </property>
         <property name="duplicatesEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>GOST</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Segmented</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Layered</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Exponential</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Anomaly</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="atmoList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>4</number>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="GostInput" name="gost">
          <layout class="QVBoxLayout" name="verticalLayout_3"/>
         </widget>
         <widget class="QWidget" name="segm">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="NsSegm">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>N_s</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="NsSegmEdit"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="hsSegm">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>h_s</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="hsSegmEdit"/>
           </item>
           <item row="2" column="1">
            <spacer name="SegmSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>73</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Layer"/>
         <widget class="ExpInput" name="exp">
          <layout class="QVBoxLayout" name="verticalLayout_2"/>
         </widget>
         <widget class="QWidget" name="anom"/>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <layout class="QVBoxLayout" name="refraction">
       <item>
        <widget class="QLabel" name="refraLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Segoe UI</family>
           <italic>true</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>Reftaction</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="refraMode">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentText">
          <string/>
         </property>
         <property name="currentIndex">
          <number>-1</number>
         </property>
         <property name="placeholderText">
          <string>Choose model...</string>
         </property>
         <property name="duplicatesEnabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Geometric round</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Geometric line</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Aware</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="refraList">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>4</number>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="line"/>
         <widget class="QWidget" name="round"/>
         <widget class="QWidget" name="aware"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ExpInput</class>
   <extends>QWidget</extends>
   <header>inputlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GostInput</class>
   <extends>QWidget</extends>
   <header>inputlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>atmoMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>atmoList</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>65</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refraMode</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>refraList</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>516</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
